components/testing/gh-actions/install_istio.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+ISTIO_VERSION="1.11.0"
```

Comment:
let's also put a `set -e` to ensure the script will error in case on of the subcommands return an exit code other than zero https://linuxhint.com/bash-set-e/

Reply:
Added `set -e`.